pkg/services/provisioning/alerting/types.go

```diff
@@ -12,23 +12,29 @@ type configVersion struct {
 
 type AlertingFile struct {
 	configVersion
+	Filename            string
 	Groups              []AlertRuleGroup
 	DeleteRules         []RuleDelete
 	ContactPoints       []ContactPoint
 	DeleteContactPoints []DeleteContactPoint
+	Policies            []NotificiationPolicy
+	ResetPolicies       []int64
```

Comment:
Can we make this strongly typed? i.e. `type OrgPolicy int64` then `ResetPolicies []OrgPolicy`? It was not immediately apparent the int64 here referred to orgID unless you really deeply understand the structure of policies already (we use int64s for other kinds of IDs too)

Reply:
Yes, good idea.